apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/util/AuthUserPasswordCheckerTest.java

```diff
@@ -63,6 +58,8 @@ public void testRegexMatch() {
 
   @Test
   public void testIsWeakPassword() {
+    AuthUserPasswordChecker checker = new AuthUserPasswordChecker(PortalConfigFaker.empty());
```

Comment:
Is it better to use mockito to handle the mock part? e.g.
```suggestion
    PortalConfig portalConfig = Mockito.mock(PortalConfig.class);
    Mockito.when(portalConfig.listOfCodeFragment()).thenReturn(Collections.emptyList());
    AuthUserPasswordChecker checker = new AuthUserPasswordChecker(portalConfig);
```

Reply:
Done.